### ISSUE TITLE
Typo in Exception 

### ISSUE THREAD

**Issue by dchabot** [NSLS-II/databroker#36]
This is just a placeholder for an oops found at 11id:

```python
In [28]: events = get_events(hdr, fields=['data'])

In [29]: list(events)
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-29-fa903d44c89f> in <module>()
----> 1 list(events)

/home/xf11id/conda_envs/collection/lib/python3.5/site-packages/databroker/databroker.py in get_events(headers, fields, fill)
    345                 raise ValueError(('{!r} field is not in any descriptor '
    346                                   'of run start {rs.uid}').
--> 347                                  format(k, header['run_star']))
    348 
    349     for header in headers:

KeyError: 'run_star'
```

Definitely not a "run_star"...

**Comment by ericdill** [comment]
This was fixed by #32. Think we could install the master branch on CHX?

**Comment by tacaswell** [comment]
closing as fixed already.
